### PR TITLE
Translate docs/helm/helm_env in Korean

### PR DESCRIPTION
**Pull request by bluefriday**
Signed-off-by: bluefriday <bluefriday86@gmail.com>

Translate docs/helm/helm_env in Korean
- https://helm.sh/docs/helm/helm_env/

### CODE REVIEW

**Comment by bluefriday** [comment]
@jmnote  @singlemocha  리뷰 부탁드립니다.

File: content/ko/docs/helm/helm_env.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+## helm env
+
+헬름 클라이언트 환경변수 정보
+
+### 개요
+
+
+Env 는 헬름에서 사용중인 모든 환경변수 정보를 출력한다.
```
Comment (jmnote) [inline content/ko/docs/helm/helm_env.md:12]:
```suggestion
Env 는 헬름에서 사용 중인 모든 환경변수 정보를 출력한다.
```

File: content/ko/docs/helm/helm_env.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+```
+  -h, --help   env 에 대한 도움말
+```
+
+### 상위 명령에서 상속된 옵션들
+
+```
+      --add-dir-header                   이 값이 참이면, 헤더에 파일 디렉토리를 추가
+      --alsologtostderr                  표준 오류를 로그 및 파일로 표시
+      --debug                            상세 내용 표시 활성화
```
Comment (jmnote) [inline content/ko/docs/helm/helm_env.md:30]:
```suggestion
      --debug                            장황한(verbose) 출력 활성화
```

File: content/ko/docs/helm/helm_env.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+helm env [flags]
+```
+
+### 옵션
+
+```
+  -h, --help   env 에 대한 도움말
+```
+
+### 상위 명령에서 상속된 옵션들
```
Comment (jmnote) [inline content/ko/docs/helm/helm_env.md:25]:
```suggestion
### 부모 명령어에서 상속된 옵션들
```

File: content/ko/docs/helm/helm_env.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      --log-dir string                   이 값이 비어있지 않을 경우, 이 값에 지정된 디렉토리에 로그 파일 쓰기 수행
+      --log-file string                  이 값이 비어있지 않을 경우, 이 값에 지정된 파일명으로 로그 파일 쓰기 수행
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가 바이트이며 값이 0일 경우 최대 파일크기 제한 없음(기본 값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 : true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
+      --repository-config string         저장소 이름 및 URL 을 포함하는 파일 경로 (기본값 "~/.config/helm/repositories.yaml")
+      --skip-headers                     이 값이 참이면, 로그파일에서 헤더 접두사를 미사용
+      --skip-log-headers                 이 값이 참이면, 로그 파일을 열 때 헤더 제외
```
Comment (jmnote) [inline content/ko/docs/helm/helm_env.md:45]:
```suggestion
      --skip-headers                     이 값이 true이면, 로그파일에서 헤더 접두어를 미사용
      --skip-log-headers                 이 값이 true이면, 로그파일을 열 때 헤더 제외
```

File: content/ko/docs/helm/helm_env.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+      --debug                            상세 내용 표시 활성화
+      --kube-apiserver string            쿠버네티스 API 서버의 주소 및 포트
+      --kube-context string              사용할 kubeconfig 컨텍스트 이름
+      --kube-token string                인증에 사용될 문자열 전달 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   이 값이 비어있지 않을 경우, 이 값에 지정된 디렉토리에 로그 파일 쓰기 수행
+      --log-file string                  이 값이 비어있지 않을 경우, 이 값에 지정된 파일명으로 로그 파일 쓰기 수행
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가 바이트이며 값이 0일 경우 최대 파일크기 제한 없음(기본 값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 : true)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_env.md:39]:
```suggestion
      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
```

File: content/ko/docs/helm/helm_env.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
+      --repository-config string         저장소 이름 및 URL 을 포함하는 파일 경로 (기본값 "~/.config/helm/repositories.yaml")
+      --skip-headers                     이 값이 참이면, 로그파일에서 헤더 접두사를 미사용
+      --skip-log-headers                 이 값이 참이면, 로그 파일을 열 때 헤더 제외
+      --stderrthreshold severity         stderr로 로그가 변경될 수 있는 최저 임계점 (기본값 2)
+  -v, --v Level                          로그 수준 상세표시 레벨
+      --vmodule moduleSpec               파일로 필터링 된 로깅을 위한 패턴=N 설정의 쉼표로 구분된 리스트
```
Comment (jmnote) [inline content/ko/docs/helm/helm_env.md:48]:
```suggestion
      --vmodule moduleSpec               파일로 필터링된 로깅을 위한 패턴=N 설정의 쉼표로 구분된 리스트
```

File: content/ko/docs/helm/helm_env.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+### 상위 명령에서 상속된 옵션들
+
+```
+      --add-dir-header                   이 값이 참이면, 헤더에 파일 디렉토리를 추가
+      --alsologtostderr                  표준 오류를 로그 및 파일로 표시
+      --debug                            상세 내용 표시 활성화
+      --kube-apiserver string            쿠버네티스 API 서버의 주소 및 포트
+      --kube-context string              사용할 kubeconfig 컨텍스트 이름
+      --kube-token string                인증에 사용될 문자열 전달 토큰
```
Comment (jmnote) [inline content/ko/docs/helm/helm_env.md:33]:
```suggestion
      --kube-token string                인증에 사용될 베어러(bearer) 토큰
```

File: content/ko/docs/helm/helm_env.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   이 값이 비어있지 않을 경우, 이 값에 지정된 디렉토리에 로그 파일 쓰기 수행
+      --log-file string                  이 값이 비어있지 않을 경우, 이 값에 지정된 파일명으로 로그 파일 쓰기 수행
```
Comment (jmnote) [inline content/ko/docs/helm/helm_env.md:37]:
```suggestion
      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
```

File: content/ko/docs/helm/helm_env.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가 바이트이며 값이 0일 경우 최대 파일크기 제한 없음(기본 값 1800)
```
Comment (jmnote) [inline content/ko/docs/helm/helm_env.md:38]:
```suggestion
      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
```

File: content/ko/docs/helm/helm_env.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+      --kube-token string                인증에 사용될 베어러(bearer) 토큰
+      --kubeconfig string                kubeconfig 파일 경로
+      --log-backtrace-at traceLocation   로깅 시 N 행에 걸친 스택 추적 내용을 표시 (기본값 :0)
+      --log-dir string                   값을 지정하면, 지정한 디렉토리에 로그 파일 기록
+      --log-file string                  값을 지정하면, 지정한 로그 파일 사용
+      --log-file-max-size uint           로그파일이 증가할 수 있는 최대 크기 지정. 단위는 메가바이트이다. 값이 0이면, 최대 파일크기는 무제한. (기본값 1800)
+      --logtostderr                      로그를 파일이 아닌 표준 출력으로 표시 (기본값 true)
+  -n, --namespace string                 요청에 대한 네임스페이스 지정
+      --registry-config string           레지스트리 구성 파일에 대한 경로 (기본값 "~/.config/helm/registry.json")
+      --repository-cache string          캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/snap/code/common/.cache/helm/repository")
```
Comment (bridgetkromhout) [inline content/ko/docs/helm/helm_env.md:42]: Same concern (this is not the same as in English) - https://github.com/helm/helm-www/blame/master/content/en/docs/helm/helm_env.md#L42

Comment (singlemocha) [reply content/ko/docs/helm/helm_env.md:42]: Good catch!  This needs to be changed as below;
캐시된 저장소 색인이 포함된 파일의 경로 (기본값 "~/.cache/helm/repository")

**Review by singlemocha — COMMENTED** [review]
Fix is needed for commented string

File: content/ko/docs/helm/helm_env.md (hunk 10 above)
Comment (bluefriday) [reply content/ko/docs/helm/helm_env.md:42]: I will modify every I PRed as your comment like issue #810 :D

**Comment by bluefriday** [comment]
> Fix is needed for commented string

@singlemocha @bridgetkromhout 
Thanks. I modified my translate as your comment.

**Review by bridgetkromhout — APPROVED** [review]
LGTM!

**Pull request merged**